### PR TITLE
Add the  Deployment and Detailed Status 

### PR DESCRIPTION
**Pull request by ojrb**
As requested, I add to the document the Deployment and Detailed status with the definition of each one. This PR solved the Issue #1805. @mypil please review and merge !

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 1]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
  
 The **Latest Security Update Status** and **Previous Security Update Status** tiles are stacked to form one blade. The **Latest Security Update Status** provides a visualization of the different deployment states devices are in regarding the latest update for each build (or version) of Windows 10, along with the revision of that update. The **Previous Security Update Status** blade provides the same information without the accompanying visualization. 
 
+The deployment and detailed states of an update, are described as follows:
+
+**Deployment States**
+* **Unknown:** Occurs when devices are not scanned for an update in some time.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Unknown** occurs when devices have not been scanned for an update for some time.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 2]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **In Progress:** When an update is applying to a specific device.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **In Progress** means an update is being applied to a specific device.
```

File: windows/deployment/update/update-compliance-security-update-status.md (hunk 1 above)
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Unknown**: Occurs when devices have not been scanned for an update for some time.
```

File: windows/deployment/update/update-compliance-security-update-status.md (hunk 2 above)
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **In Progress**: When an update is being applied to a specific device.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 3]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Progress Stalled:** When for any reason, the progress of an update stops, for example caused by netowork problems.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Progress Stalled**: When the progress of an update stops for any reason, such as network failure.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 4]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Failed:** When an update, failed to apply to a specific device.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Failed**: When the update process for a specific device is unsuccessful.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 5]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Update Completed:** When an update applies to a device without problems.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Update Completed**: When a device is updated successfully.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 6]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Deferred:** When the application of an update for a device, it is deferred for another time. For example postponing the application one day after planned.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Deferred**: A device update has been postponed to a later time. 
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 7]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+The deployment and detailed states of an update, are described as follows:
+
+**Deployment States**
+* **Unknown:** Occurs when devices are not scanned for an update in some time.
+* **In Progress:** When an update is applying to a specific device.
+* **Progress Stalled:** When for any reason, the progress of an update stops, for example caused by netowork problems.
+* **Failed:** When an update, failed to apply to a specific device.
+* **Update Completed:** When an update applies to a device without problems.
+* **Deferred:** When the application of an update for a device, it is deferred for another time. For example postponing the application one day after planned. 
+* **Cancelled:** When the applying of an update is cancelled by the user.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Cancelled**: Indicates that an update has been cancelled by the user.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 8]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+
+**Detailed States**
+* **Dowload Started:** Indicates that downloading the contents of a specific update started.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Download Started**: Indicates that a specific update has started to download.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 9]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Download Succeeded:** Indicates that downloading the content of a specific update ended successfully.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Download Succeeded**: Indicates that the download of a specific update has been successful.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 10]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Download Hand Off:** Indicates that you can continue downloading an update at a point in time where it was stopped.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Download Hand Off**: Indicates that you can continue downloading an update at the point where it was stopped.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 11]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Install Hand Off:** Indicates that you can continue installing an update at a point in time where it was stopped.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Install Hand Off**: Indicates that you can continue installing an update at the point where it was stopped.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 12]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Reboot Pending:** Indicates that an update needs a reboot to finish applying.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Reboot Pending**: Indicates that you need reboot the device to complete the update process.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 13]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Service Stack Downloaded:** Indicates that the Service Stack Update (The component that installs Windows update) is downloading.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Service Stack Downloaded**: Indicates that the Servicing Stack Update (the component that installs the Windows update) is downloading.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 14]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Update Offered:** Indicates that an update is offered, being optional the application of the same.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Update Offered**: Indicates that an optional update is available.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 15]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
...
+* **Update Detected:** Indicates that a needed update is detected to apply.
```
Comment (nenonix) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Update Detected**: Indicates that a necessary update is available.
```

File: windows/deployment/update/update-compliance-security-update-status.md [hunk 16]
```diff
@@ -21,6 +21,27 @@ The **Overall Security Update Status** blade provides a visualization of devices
  
 The **Latest Security Update Status** and **Previous Security Update Status** tiles are stacked to form one blade. The **Latest Security Update Status** provides a visualization of the different deployment states devices are in regarding the latest update for each build (or version) of Windows 10, along with the revision of that update. The **Previous Security Update Status** blade provides the same information without the accompanying visualization. 
 
+The deployment and detailed states of an update, are described as follows:
```
Comment (JohanFreelancer9) [inline windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
The deployment and detailed states of an update are described as follows:
```

File: windows/deployment/update/update-compliance-security-update-status.md (hunk 12 above)
Comment (JohanFreelancer9) [reply windows/deployment/update/update-compliance-security-update-status.md]:
```suggestion
* **Reboot Pending**: Indicates that you need to reboot the device to complete the update process.
```

**Comment by mypil** [comment]
@ojrb  - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ojrb** [comment]
Ready @mypil  I accept al the reviews of @nenonix and @JohanFreelancer9

**Comment by mypil** [comment]
@jaimeo - Can you please assist to check, approve and merge the content of this PR based on issue #1805?

Thank you.

**Comment by jaimeo** [comment]
I would like the PM/SME for this, @cinglis-msft to review and approve prior to merging. Also, use sentence case, not title case, (therefore "Detailed status" not "Detailed Status").

**Comment by chinglis** [comment]
Please let me review, I don't think some of those states even exist anymore and am not sure if those descriptions are accurate -- their being surfaced in the UX in the first place was unintentional. I need to confirm with devs.

**Comment by ojrb** [comment]
@mypil I modified the tittle case to sentence case as requested by @jaimeo, please review and merge !

**Comment by cinglis-msft** [comment]
FYI I’ve not yet had time to review w/ devs, we have been going thru an office move and planning. I hope to review today.

Charles

On Jun 4, 2019, at 5:43 AM, Orlando Rodriguez <notifications@github.com<mailto:notifications@github.com>> wrote:


@mypil<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmypil&data=02%7C01%7C%7C10ebd702109c45b1a28b08d6e8ea4d38%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636952490339714949&sdata=pfC2awfvxfBkBaLA57mzibPKFQzG2Ympodfh9hd9kQ8%3D&reserved=0> I modified the tittle case to sentence case as requested by @jaimeo<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjaimeo&data=02%7C01%7C%7C10ebd702109c45b1a28b08d6e8ea4d38%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636952490339734989&sdata=syeRGDN23WiNM6CBj8H7ISNWgF8Hfqt4ur4HTuwBMNY%3D&reserved=0>, please review and merge !

**Comment by cinglis-msft** [comment]
I have sent a mail to @jaimeo with all possible statuses/states and a definition for each.

**Comment by mypil** [comment]
@jaimeo - Please advise what changes you would like @ojrb to add to this PR based on what @cinglis-msft shared with you. Thank you.

**Comment by jaimeo** [comment]
Thanks for the contribution--the content idea is good, but it's going to be simplest for me to edit the material from @cinglis-msft and merge that as a separate PR. So I'm closing this one.

**Pull request closed without merging**